manifests/mysql_connector.pp

```diff
@@ -0,0 +1,19 @@
+class confluence::mysql_connector (
+  String $version      = $confluence::mysql_connector_version,
+  String $installdir   = $confluence::mysql_connector_install,
+) {
+
```

Comment:
this is supposed to be a private class that won't get called from other modules, only from the init.pp, right? In this case, you can add https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/REFERENCE.md#assert_private
```suggestion
assert_private()
```

Reply:
Yes I don't see any reason that soemone would want to call this class directly.